<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1172</width>
    <height>561</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>435</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>PMS</string>
  </property>
  <property name="windowIcon">
   <iconset resource="pms_resources.qrc">
    <normaloff>:/images/PMS_icon.png</normaloff>:/images/PMS_icon.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="sizeConstraint">
     <enum>QLayout::SetMaximumSize</enum>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="frame">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>90</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>90</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QFrame {
	background-image: url(:/images/banner.png);
	background-repeat:no-repeat;
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QFrame" name="frame_16">
         <property name="layoutDirection">
          <enum>Qt::RightToLeft</enum>
         </property>
         <property name="styleSheet">
          <string notr="true">background-image: url(:&quot;&quot;);</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <widget class="QLabel" name="label_12">
          <property name="geometry">
           <rect>
            <x>370</x>
            <y>16</y>
            <width>61</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ffffff;&quot;&gt;Logged in as&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_13">
          <property name="geometry">
           <rect>
            <x>370</x>
            <y>30</y>
            <width>91</width>
            <height>21</height>
           </rect>
          </property>
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt; font-weight:600; color:#ffffff;&quot;&gt;Supadoge&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_14">
          <property name="geometry">
           <rect>
            <x>370</x>
            <y>50</y>
            <width>101</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-style:italic; color:#ffffff;&quot;&gt;Supreme supervisor&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
         <widget class="QGraphicsView" name="graphicsView">
          <property name="geometry">
           <rect>
            <x>312</x>
            <y>18</y>
            <width>48</width>
            <height>48</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">border-image: url(:/images/doge.jpeg);
border: 2px solid;
border-color: rgb(17, 53, 108);
border-radius: 5px</string>
          </property>
          <property name="resizeAnchor">
           <enum>QGraphicsView::AnchorViewCenter</enum>
          </property>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_3">
         <property name="layoutDirection">
          <enum>Qt::RightToLeft</enum>
         </property>
         <property name="styleSheet">
          <string notr="true">QFrame {
	background-image: url(:&quot;&quot;);
	boder-width:-1
}

QToolButton {    
	color: rgb(255, 255, 255);
	background-image: url(:&quot;&quot;);
 }

QToolButton:hover {    
	border-image: url(:/images/banner_hovered.png);
 }

QToolButton:checked {    
	border-image: url(:/images/banner_selected.png);
 }

QToolButton:pressed {    
	border-image: url(:/images/banner_pressed.png);
 }
</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <property name="lineWidth">
          <number>0</number>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="sizeConstraint">
           <enum>QLayout::SetMinAndMaxSize</enum>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QToolButton" name="toolButton">
            <property name="minimumSize">
             <size>
              <width>90</width>
              <height>90</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>90</width>
              <height>90</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QToolButton {    
	color: rgb(255, 255, 255);
 }

QToolButton#toolButton:checked {    
	border-image: url(:/images/banner_selected.png);
 }
</string>
            </property>
            <property name="text">
             <string>System</string>
            </property>
            <property name="icon">
             <iconset resource="pms_resources.qrc">
              <normaloff>:/images/setting_tools.png</normaloff>:/images/setting_tools.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>32</width>
              <height>32</height>
             </size>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
            <property name="autoExclusive">
             <bool>true</bool>
            </property>
            <property name="toolButtonStyle">
             <enum>Qt::ToolButtonTextUnderIcon</enum>
            </property>
            <property name="autoRaise">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="toolButton_mng">
            <property name="minimumSize">
             <size>
              <width>90</width>
              <height>90</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>90</width>
              <height>90</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QToolButton {    
	color: rgb(255, 255, 255);
 }

QToolButton#toolButton:checked {    
	border-image: url(:/images/banner_selected.png);
 }
</string>
            </property>
            <property name="text">
             <string>Events</string>
            </property>
            <property name="icon">
             <iconset resource="pms_resources.qrc">
              <normaloff>:/images/calendar.png</normaloff>:/images/calendar.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>32</width>
              <height>32</height>
             </size>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
            <property name="autoExclusive">
             <bool>true</bool>
            </property>
            <property name="toolButtonStyle">
             <enum>Qt::ToolButtonTextUnderIcon</enum>
            </property>
            <property name="autoRaise">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="toolButton_rehab">
            <property name="minimumSize">
             <size>
              <width>90</width>
              <height>90</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>90</width>
              <height>90</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QToolButton {    
	color: rgb(255, 255, 255);
 }

QToolButton#toolButton:checked {    
	border-image: url(:/images/banner_selected.png);
 }
</string>
            </property>
            <property name="text">
             <string>Rehabilitation</string>
            </property>
            <property name="icon">
             <iconset resource="pms_resources.qrc">
              <normaloff>:/images/education.png</normaloff>:/images/education.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>32</width>
              <height>32</height>
             </size>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
            <property name="autoExclusive">
             <bool>true</bool>
            </property>
            <property name="toolButtonStyle">
             <enum>Qt::ToolButtonTextUnderIcon</enum>
            </property>
            <property name="autoRaise">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="toolButton_etrr">
            <property name="minimumSize">
             <size>
              <width>90</width>
              <height>90</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>90</width>
              <height>90</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QToolButton {    
	color: rgb(255, 255, 255);
 }

QToolButton#toolButton:checked {    
	border-image: url(:/images/banner_selected.png);
 }
</string>
            </property>
            <property name="text">
             <string>Entry &amp;&amp; Release</string>
            </property>
            <property name="icon">
             <iconset resource="pms_resources.qrc">
              <normaloff>:/images/door_open.png</normaloff>:/images/door_open.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>32</width>
              <height>32</height>
             </size>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
            <property name="autoExclusive">
             <bool>true</bool>
            </property>
            <property name="toolButtonStyle">
             <enum>Qt::ToolButtonTextUnderIcon</enum>
            </property>
            <property name="autoRaise">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="toolButton_Healthcare">
            <property name="minimumSize">
             <size>
              <width>90</width>
              <height>90</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>90</width>
              <height>90</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QToolButton {    
	color: rgb(255, 255, 255);
 }

QToolButton#toolButton:checked {    
	border-image: url(:/images/banner_selected.png);
 }
</string>
            </property>
            <property name="text">
             <string>Healthcare</string>
            </property>
            <property name="icon">
             <iconset resource="pms_resources.qrc">
              <normaloff>:/images/health.png</normaloff>:/images/health.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>32</width>
              <height>32</height>
             </size>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
            <property name="autoExclusive">
             <bool>true</bool>
            </property>
            <property name="toolButtonStyle">
             <enum>Qt::ToolButtonTextUnderIcon</enum>
            </property>
            <property name="autoRaise">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="toolButton_Dining">
            <property name="minimumSize">
             <size>
              <width>90</width>
              <height>90</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>90</width>
              <height>90</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string>Dining</string>
            </property>
            <property name="icon">
             <iconset resource="pms_resources.qrc">
              <normaloff>:/images/qip_eating.png</normaloff>:/images/qip_eating.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>32</width>
              <height>32</height>
             </size>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
            <property name="autoExclusive">
             <bool>true</bool>
            </property>
            <property name="toolButtonStyle">
             <enum>Qt::ToolButtonTextUnderIcon</enum>
            </property>
            <property name="autoRaise">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="toolButton_Dashboard">
            <property name="minimumSize">
             <size>
              <width>90</width>
              <height>90</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>90</width>
              <height>90</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string>Dashboard</string>
            </property>
            <property name="icon">
             <iconset resource="pms_resources.qrc">
              <normaloff>:/images/speedometer.png</normaloff>:/images/speedometer.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>32</width>
              <height>32</height>
             </size>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
            <property name="autoRepeat">
             <bool>false</bool>
            </property>
            <property name="autoExclusive">
             <bool>true</bool>
            </property>
            <property name="toolButtonStyle">
             <enum>Qt::ToolButtonTextUnderIcon</enum>
            </property>
            <property name="autoRaise">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frame_2">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="sizeConstraint">
        <enum>QLayout::SetMaximumSize</enum>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QFrame" name="frame_18">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(217, 217, 217);
</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QMdiArea" name="mdiArea">
            <property name="lineWidth">
             <number>0</number>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="pms_resources.qrc"/>
 </resources>
 <connections/>
</ui>
